plan/plan.go

```diff
@@ -148,9 +119,29 @@ func (p *Plan) Calculate() *Plan {
 	}
 
 	changes := &Changes{}
-	changes.Create = t.getCreates()
-	changes.Delete = t.getDeletes()
-	changes.UpdateNew, changes.UpdateOld = t.getUpdates()
+
+	for _, topRow := range t.rows {
```

Comment:
From my point of view it would be better to keep the methods getCreates(), getDeletes() and getUpdates() for a better maintainability.

Reply:
I wonder if the previous state was really more maintainable. The functions were all looping over the same map, now it's only one loop, thus faster and (from my point of view) more readable.